runexporterapp.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+cd ironic-prometheus-exporter/
+export IRONIC_CONFIG=/etc/ironic/ironic.conf
+export FLASK_APP=ironic_prometheus_exporter/app/exporter.py
+export FLASK_RUN_HOST=$HOST_IP
+export FLASK_RUN_PORT=5001
+python -m flask run &
```

Comment:
Let us use a real WSGI runner, like uWSGI or gunicorn. This is not intended for production and will probably behave poorly.

Reply:
Something like http://flask.pocoo.org/docs/1.0/deploying/wsgi-standalone/#gunicorn right?

Reply:
Yeah, if `gunicorn` is in CentOS (I know that `uwsgi` is).

Reply:
https://centos.pkgs.org/7/centos-extras-x86_64/python-gunicorn-18.0-2.el7.noarch.rpm.html seems we can have it installed =)